frontend/src/util/MessfaehigkeitRanking.ts

```diff
@@ -0,0 +1,42 @@
+import Fahrzeugklasse from "@/types/enum/Fahrzeugklasse";
+import ZaehldatenIntervall from "@/types/enum/ZaehldatenIntervall";
+
+export function useMessfaehigkeitRanking() {
+
+    let rankingFahrzeugklassen: Fahrzeugklasse[] = [Fahrzeugklasse.SUMME_KFZ, Fahrzeugklasse.ZWEI_PLUS_EINS, Fahrzeugklasse.ACHT_PLUS_EINS];
```

Comment:
Direkt in der Fahrzeugklasse.ts anlegen und dann hier nur noch als const verwenden

Reply:
done